New doc about getting useful information when reporting problems.

patrickvacek opened the pull request:
This came out of a discussion with our support team.

tkfu commented on docs/ota-client-guide/modules/ROOT/pages/reporting-problems.adoc:
```diff
@@ -0,0 +1,60 @@
+
+=== aktualizr version
+
+On an active device accessible via the commandline, enter `aktualizr --version`. Otherwise, you can examine the link:https://github.com/advancedtelematic/meta-updater/blob/master/recipes-sota/aktualizr/aktualizr_git.bb[aktualizr recipe in meta-updater] in your build environment and look for the `SRCREV` variable.
```
```suggestion
On an active device accessible via the commandline, enter `aktualizr --version`. Otherwise, you can examine the aktualizr recipe in meta-updater in your build environment and look for the `SRCREV` variable. This recipe will normally be located (relative to your Yocto build directory) at `../meta-updater/recipes-sota/aktualizr/aktualizr_git.bb`.
```
As much as I'd like to hope that people will follow this instruction correctly, I strongly suspect that they'll follow your link to the latest master version of `aktualizr_git.bb` and report the version there, instead of looking at their local copy.

patrickvacek commented:
> As much as I'd like to hope that people will follow this instruction correctly, I strongly suspect that they'll follow your link to the latest master version of `aktualizr_git.bb` and report the version there, instead of looking at their local copy.

Yeah, fair point. I had that thought, too. I've incorporated your suggestion and squashed the commit.
